src/plugins/intel_cpu/tests/functional/single_layer_tests/normalize.cpp

```diff
@@ -90,7 +90,7 @@ class NormalizeL2LayerCPUTest : public testing::WithParamInterface<NormalizeL2La
             ov::Tensor tensor;
             if (funcInput.get_element_type().is_real()) {
                 tensor = ov::test::utils::create_and_fill_tensor(
-                        funcInput.get_element_type(), targetInputStaticShapes[i], 10, -5, 7, 222);
+                        funcInput.get_element_type(), targetInputStaticShapes[i], 5, 0, 7, 222);
```

Comment:
After changes in fill_data_random, this test start to fail. So have to fix it as well